stable/fluent-bit/templates/config.yaml

```diff
@@ -14,7 +14,6 @@ data:
         Flush        {{ .Values.service.flush }}
         Daemon       Off
         Log_Level    {{ .Values.service.logLevel }}
-        Parsers_File parsers.conf
```

Comment:
Is there not a parsers.conf in the image any more?

Reply:
In [config.yaml:L161 ff](https://github.com/helm/charts/blob/master/stable/fluent-bit/templates/config.yaml#L161) all settings are written to the subPath `parsers.conf`

The final mount for the [PARSER]s configuration is done in [daemonset.yaml:L95](https://github.com/helm/charts/blob/master/stable/fluent-bit/templates/daemonset.yaml#L95)
```Yaml
{{- if .Values.parsers.enabled }}
        - name: config
          mountPath: /fluent-bit/etc/parsers_custom.conf
          subPath: parsers.conf
{{- end }}
```
There is the condition `{{- if .Values.parsers.enabled }}` *and* then the final mount will be `parsers_custom.conf` inside the container.

So there is no need to have the above line for the [SERVICE] section as there will not be a file `parsers.conf` inside the container :)